components/crud-web-apps/volumes/Makefile

```diff
@@ -1,11 +1,9 @@
-IMG ?= gcr.io/arrikto-playground/kubeflow/volumes-web-app
-TAG ?= $(shell git describe --tags)
-DOCKERFILE=volumes/Dockerfile
+IMG ?= public.ecr.aws/j1r0q0g6/notebooks/volumes-web-app
+TAG ?= $(shell git describe --tags --always)
+DOCKERFILE ?= volumes/Dockerfile
 
 docker-build:
-	cp Dockerfile.dockerignore ../.dockerignore
-	-cd ../ && docker build -t ${IMG}:${TAG} -f ${DOCKERFILE} .
-	rm ../.dockerignore
+	cd ../ && docker build -t ${IMG}:${TAG} -f ${DOCKERFILE} .
```

Comment:
This part is a bit weird. I mean mentioning paths as if we are in a different directory because we will be using them in a different context (after `cd`ing).

May I suggest we
1. Drop the `DOCKERFILE` variable (I think we shouldn't be messing with it) OR make it relative to this directory
   I.e.,
   ```
   DOCKERFILE=Dockerfile
   ```
2. Drop the `cd`ing part from the Makefile target and use the parent directory as the path for the Docker building context (https://docs.docker.com/engine/reference/commandline/build/#build-with-path)
   This will make the Makefile target as follows
   ```Makefile
   docker-build:
       docker build -t ${IMG}:${TAG} -f Dockerfile ..  # Or "-f ${DOCKERFILE}" depending on the decision of (1)
   ```

I understand the reason we are `cd`ing to the parent directory is to use a single `.dockerignore` file for all the web apps.
I tested locally that we maintain this behavior with the change I proposed, so there should be no problem. Let's also have others verify this as well (you, the CI, other?).

Then, we can follow up changing this for the other web apps as well (this is not a blocker for the release of course).

Reply:
@elikatsis ACK, this is a very good suggestion!

I'll add a commit for Volumes web app in this PR, since it's fixing the formatting for this component. I'll then open up a new issue to keep track of the uniformity in the dockerfiles for the other web apps as well.